components/feature/top-sites/src/main/java/mozilla/components/feature/top/sites/DefaultTopSitesStorage.kt

```diff
@@ -100,4 +204,24 @@ class DefaultTopSitesStorage(
 
         return topSites
     }
+
+    companion object {
```

Comment:
There's probably a better place to put these than in a companion. I would ask someone on AC what their preference would be